src/js/claims-status/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "appName": "Claims Status",
+  "entryPoint": "./src/js/claims-status/claims-status-entry.jsx",
+  "bundleName": "claims-status",
+  "rootUrl": "/track-claims",
+  "production": true
+}
```

Comment:
So with the `entryPoint` property, we could ditch the giant hardcoded `entryFiles` array in the webpack config?

Reply:
I think so, yes.